Formula/siril.rb

```diff
@@ -56,7 +57,8 @@ def install
 
     # siril uses pkg-config but it has wrong include paths for several
     # headers. Work around that by letting it find all includes.
-    ENV.append_to_cflags "-I#{HOMEBREW_PREFIX}/include -Xpreprocessor -fopenmp -lomp"
+    ENV.append_to_cflags "-I#{HOMEBREW_PREFIX}/include"
+    ENV.append_to_cflags "-Xpreprocessor -fopenmp -lomp" if OS.mac?
```

Comment:
Original logic caused linkage to both `libomp.so` and `libgomp.so` so need new revision to match dependencies:
```
Dynamic Section:
  NEEDED       libomp.so
  ...
  NEEDED       libgomp.so.1
...
Version References:
  ...
  required from libomp.so:
    0x0aa77e6e 0x00 08 VERSION
```